### DISCUSSION TITLE
filter by image availability

### DISCUSSION THREAD

**Discussion by thea-m** [SChAth/ScAthiop#1845] [Q&A] [answered]
I would like to find all records of scrolls for which images or links to images are available. What would be the best way to do this? In the faceted search, it is possible to filter results by image availability, but since I am not searching for any terms that doesn't help me here. No such filter is available in the manuscript search, as far as I can see. Thank you!

**Comment by PietroLiuzzo** [comment, marked as answer]
Dear @thea-m I think the answer to this is the following search, at the moment.
https://betamasaheft.eu/facet.html?query=manuscript&type-facet=manuscript&form-facet=Scroll&images-facet=yes
notice the trick: because we are indexing publication statement, "manuscript" will always occur as text. 
now, the need for this workaround will disappear with the imminent release, where searching without text will be possible.
the correctness of the result can be confirmed running a more explicit Xpath search
https://betamasaheft.eu/xpath?xpath=collection%28%24config%3Adata-rootMS%29%2F%2Ft%3ATEI%5Bdescendant%3A%3At%3Aidno%5B%40facs%5D%5B%40n%5D%5D%5Bdescendant%3A%3At%3AobjectDesc%5B%40form%3D%22Scroll%22%5D%5D

**Reply by thea-m** [reply]
Perfect, I'll remember that!
